spacy_transformers/layers/transformer_model.py

```diff
@@ -216,10 +214,11 @@ def _convert_transformer_inputs(model, wps: WordpieceBatch, is_train):
 
 def _convert_transformer_outputs(model, inputs_outputs, is_train):
     _, model_output = inputs_outputs
+    tensors = tuple(model_output[k] for k in model_output.keys() if k not in EXCLUDED_KEYS)
 
     def backprop(d_model_output: ModelOutput) -> ArgsKwargs:
         return ArgsKwargs(
-            args=(model_output.last_hidden_state,),
+            args=(tensors,),
```

Comment:
This is the main part where I found divergence between the current code and before #283: before, the `tensors` would include more than just the last hidden state, e.g. for `roberta-base` there would also be a `pooler_output` output that would get gradients too.